src/components/lib/posix/posix.c

```diff
@@ -198,24 +197,30 @@ cos_tkill(int tid, int sig)
 	return 0;
 }
 
+static inline microsec_t
+time_to_microsec(const struct timespec *t)
+{
+	time_t seconds    		= t->tv_sec;
+	long nano_seconds 		= t->tv_nsec;
+	microsec_t microseconds = seconds * 1000000 + nano_seconds / 1000;
+	return microseconds;
```

Comment:
separate declarations from `return`. Also, spaces in declaration needs to be fixed.